plugins/gatsby-source-install-config/gatsby-node.js

```diff
@@ -0,0 +1,159 @@
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions;
+
+  createTypes(`
+    type InstallConfig @dontInfer {
+      id: ID!
+      agentName: String!
+      agentType: String!
+      title: String!
+      intro: IntroFile
+      appInfo: [AppInfoOption!]!
+      steps: [InstallStep]
+      mdxFiles: [Mdx]
+      whatsNextMdx: String!
+    }
+    type IntroFile @dontInfer {
+      filePath: String
+      mdx: Mdx
+    }
+    type AppInfoOption @dontInfer {
+      optionType: String!
+      label: String!
+      options: [AppInfoOptionValue!]!
+    }
+    type AppInfoOptionValue @dontInfer {
+      value: String!
+      displayName: String!
+      recommendedGuided: Boolean
+    }
+    type InstallStep @dontInfer {
+      filePath: String
+      mdx: Mdx
+      overrides: [StepOverride]
+    }
+    type StepOverride @dontInfer {
+      optionType: String!
+      overrideConfig: [StepOverrideConfig]
+    }
+    type StepOverrideConfig @dontInfer {
+      value: String!
+      filePath: String
+      mdx: Mdx
+      skip: Boolean
+      overrides: [StepOverride]
+    }
+    type Mdx implements Node {
+      frontmatter: Frontmatter
+    }
+    type Frontmatter {
+      componentType: String! 
+      headingText: String
+      inputOptions: [InputOption]
+      tipMdx: String
+      agentConfigFilePath: String
+      optionType: String
+    }
+    type InputOption @dontInfer {
+      labelMdx: String!
+      value: String! 
+      infoMdx: String! 
```

Comment:
This would update things for the previous comment
```suggestion
      name: String!
      codeLine: String!
      label: String!
      defaultValue: String!
      toolTip: String,
      url: InputUrl,
    }
    type InputUrl {
      title: String!
      href: String!
```